bindings/wasm/examples/worker.ts

```diff
@@ -797,8 +797,41 @@ async function exportModels(defaults: GlobalDefaults, manifold?: Manifold) {
       [zipFile],
       {type: 'application/vnd.ms-package.3dmanufacturing-3dmodel+xml'});
 
-  self.postMessage({
+  return {
     glbURL: URL.createObjectURL(blobGLB),
     threeMFURL: URL.createObjectURL(blob3MF)
-  });
+  }
+  // self.postMessage({
+  //   glbURL: URL.createObjectURL(blobGLB),
+  //   threeMFURL: URL.createObjectURL(blob3MF)
+  // });
+}
+
+function evaluateCADToManifold(code: string) {
+  const footerCode = `
+if (typeof result !== "undefined") {
+  return result;
+}
+`;
+  const globalDefaults = {} as GlobalDefaults;
+  const context = {
+    globalDefaults,
+    exportModels,
+    glMatrix,
+    module,
+    ...Object.fromEntries(
+      exposedFunctions.map((name) => [name, (module as any)[name]]),
+    ),
+  };
+  const evalFn = new Function(
+    ...Object.keys(context),
+    code + ";\n" + footerCode,
+  );
+  const manifold = evalFn(...Object.values(context));
+  return { globalDefaults, manifold };
+}
+
+export async function evaluateCADToModel(code: string) {
```

Comment:
I like this because we can probably publish this to npm, as it's more general purpose than our worker was.